databricks/koalas/ml.py

```diff
@@ -14,13 +14,24 @@
 # limitations under the License.
 #
 
-import pandas as pd
+from typing import List, Tuple, TYPE_CHECKING
+
 import numpy as np
+import pandas as pd
+import pyspark
+
 from pyspark.ml.feature import VectorAssembler
 from pyspark.ml.stat import Correlation
 
 
-def corr(kdf, method='pearson'):
+if TYPE_CHECKING:
+    import databricks.koalas as ks
+
+
+CORRELATION_OUTPUT_COLUMN = '_correlation_output'
```

Comment:
it is good style to prefix it with `_koalas` to make sure it does not clash

Reply:
Fair enough. I might change it to `_koalas_corr` in a future pull request.